plugin-server/src/utils/db/postgres.ts

```diff
@@ -3,11 +3,60 @@
 import { StatsD } from 'hot-shots'
 import { Client, Pool, PoolClient, QueryConfig, QueryResult, QueryResultRow } from 'pg'
 
+import { PluginsServerConfig } from '../../types'
 import { instrumentQuery } from '../../utils/metrics'
+import { status } from '../status'
+import { createPostgresPool } from '../utils'
 import { POSTGRES_UNAVAILABLE_ERROR_MESSAGES } from './db'
 import { DependencyUnavailableError } from './error'
 
-export function postgresQuery<R extends QueryResultRow = any, I extends any[] = any[]>(
+export enum PostgresUse {
+    COMMON, // Main PG master with common tables, we need to move as many queries away from it as possible
```

Comment:
maybe say `COMMON_WRITE` / `COMMON_MASTER` and move `COMMON_READ` to be the first one <- so folks more likely pick the read one for new things